# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dansguardian/gcc44.patch
# Copyright (C) 2010 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nru dansguardian-2.10.1.1.orig/src/ConnectionHandler.cpp dansguardian-2.10.1.1/src/ConnectionHandler.cpp
--- dansguardian-2.10.1.1.orig/src/ConnectionHandler.cpp        2009-02-25 12:36:22.000000000 +0100
+++ dansguardian-2.10.1.1/src/ConnectionHandler.cpp     2009-07-05 01:15:52.000000000 +0200
@@ -44,6 +44,7 @@
 #include <istream>

 #ifdef ENABLE_ORIG_IP
+#include <limits.h>
 #include <linux/types.h>
 #include <linux/netfilter_ipv4.h>
 #endif
diff -Nru dansguardian-2.10.1.1.orig/src/downloadmanagers/fancy.cpp dansguardian-2.10.1.1/src/downloadmanagers/fancy.cpp
--- dansguardian-2.10.1.1.orig/src/downloadmanagers/fancy.cpp   2008-11-18 12:27:04.000000000 +0100
+++ dansguardian-2.10.1.1/src/downloadmanagers/fancy.cpp        2009-07-05 01:15:12.000000000 +0200
@@ -26,6 +26,7 @@
 #include "../HTMLTemplate.hpp"
 #include "../ConnectionHandler.hpp"

+#include <stdio.h>
 #include <syslog.h>
 #include <sys/time.h>
 #include <sys/types.h>

